src/libs/ReportUtils.ts

```diff
@@ -8197,6 +8197,33 @@ function isEmptyReport(report: OnyxEntry<Report>, isReportArchived = false): boo
     return generateIsEmptyReport(report, isReportArchived);
 }
 
+/**
+ * Checks if there are any empty (no money) open expense reports for a specific policy and user.
+ * An empty report is defined as having total === 0 and nonReimbursableTotal === 0.
```

Comment:
Hm, interesting. I would've thought it would be based on the presence of transactions. Couldn't a report have a total of 0 even if it had transactions? E.g. a purchase and then a return of that purchase.